docs/installation/server-config/security.rst

```diff
@@ -55,6 +58,22 @@ Configuration KSQL for Secured Apache Kafka clusters
 
 The following are common configuration examples.
 
+.. _config-security-ssl:
+
+Configuring Kafka Encrypted Communication
+-----------------------------------------
+
+This configuration enables KSQL to connect to a Kafka cluster over SSL, with a user supplied trust store:
+
+.. code:: bash
+    security.protocol=SSL
+    ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+    ssl.truststore.password=confluent
+
+The exact settings you will need will vary depending on the security settings of the Kafka brokers,
```

Comment:
suggest: `The exact settings required will vary...`

Reply:
done